lib/sensors/low_power_current.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <core/types.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+// The datasheet lists four possible I2C addresses; 0x40 has been defined as the default
+constexpr std::uint8_t kDefaultI2cAddress = 0x40;
+constexpr std::uint8_t kI2cAddress2       = 0x41;
+constexpr std::uint8_t kI2cAddress3       = 0x44;
+constexpr std::uint8_t kI2cAddress4       = 0x45;
```

Comment:
`LowPowerAddressX` where x in [2,4]

Reply:
These are defined purely for the sake of documenting what the addresses are from the datasheet. They aren't actually used anywhere else in the code. Therefore, should these be here? Or are they completely redundant and should be removed?